### DISCUSSION TITLE
关闭按钮点击没反应

### DISCUSSION THREAD

**Discussion by letoinfo** [he3-app/he3-feedback#130] [Bug Report]
每次关闭再打开He3后，第1次点击右上角关闭可以正常关掉窗口，无论是快捷键还是点击任务栏右下角再次显示He3窗口后，点击右上角关闭没有任何反应，双击右上角关闭可以最大化或还原窗口，看样子只是个图片贴在那里，没有绑定事件。

**Comment by lyzhang1999** [comment]
请问是 windows 系统吗？

**Reply by letoinfo** [reply]
是的，之前 Windows 10 和 Windows 11 都出现这个情况。
刚才我都重新安装了下，Windows 11 机器不再出现这种情况了，但 Windows 10 还是不行

**Comment by lyzhang1999** [comment]
感谢反馈，我已经建了一个 issues 来跟踪这个问题：https://github.com/he3-app/he3-feedback/issues/133
